Cargo.toml

```diff
@@ -18,7 +18,9 @@ anyhow = "1.0.58"
 async-trait = "0.1.57"
 borsh = "0.9.3"
 bs58 = "0.4.0"
-bundlr-sdk = { version = "0.3.0", default-features = false, features = ["solana"] }
+bundlr-sdk = { version = "0.3.0", default-features = false, features = [
```

Comment:
Was the formatting intentional?

Reply:
It's autoformatting when I run `cargo clippy` on file save. I'm not sure why it's breaking the line here, at line 86, instead of the max_width of 100 set in the rustfmt file.